Duplicati/Library/Snapshots/WindowsSnapshot.cs

```diff
@@ -285,15 +265,8 @@ public override string ConvertToSnapshotPath(string localPath)
 
             var root = SystemIO.IO_WIN.GetPathRoot(localPath);
             var volumePath = _vssBackupComponents.GetVolumeFromCache(root);
-
-            // Note: Do NOT use Path.Combine as it strips the UNC path prefix
             var subPath = localPath.Substring(root.Length);
-            if (!subPath.StartsWith(Util.DirectorySeparatorString, StringComparison.Ordinal))
-            {
-                volumePath = Util.AppendDirSeparator(volumePath, Util.DirectorySeparatorString);
-            }
-
-            var mappedPath = volumePath + subPath;
+            var mappedPath = Path.Combine(volumePath, subPath);

```

Comment:
Would it be safer to use `SystemIO.IO_WIN.PathCombine` instead of `Path.Combine` here?

Reply:
Thanks.  I've pushed a fix.